GameData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Data of application.
+    /// </summary>
+    public class GameData
+    {
+        readonly public int[,] data;
+        readonly public int[,] newData;
+
+        /// <summary>
+        /// Data of application.
+        /// </summary>
+        public GameData()
+        {
+            int column = UserInterface.GetValueInRange("Input amount of columns", 5, 200);
```

Comment:
string values should be saved as constants

Reply:
Changed values to constants.